www/src/components/Features.tsx

```diff
@@ -1,30 +1,53 @@
 import React, { FC } from 'react';
-import { FiBriefcase, FiLock, FiZap } from 'react-icons/fi';
+import { FaBatteryFull, FaSeedling } from 'react-icons/fa';
+import { FiBriefcase, FiLock, FiTerminal, FiZap } from 'react-icons/fi';
 
 const features = [
   {
     title: 'Automatic typesafety',
     description:
-      'Automatic typesafety & autocompletion inferred from your API-paths, their input data, & outputs.',
+      'Made a server side change? Typescript will warn you client side! This is inferred from your API-paths, their inputs & outputs.',
```

Comment:
```suggestion
      'Made a server-side change? Typescript will immediately throw errors on the client before you even save the file! This is inferred from your API-paths, their inputs & outputs.',
```

maybe "immediately throw errors" is a bit aggressive 🤔

Reply:
how about

`Instant feedback with type errors`

Reply:
```suggestion
      'Made a server-side change? Typescript will instantly show red squiggly lines on the client before you even save the file! This is inferred from your API-paths, their inputs & outputs.',
```